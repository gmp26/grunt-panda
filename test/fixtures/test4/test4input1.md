````
title: test4input1
````

# input1
## subtitle
_emphasis_
__bold__