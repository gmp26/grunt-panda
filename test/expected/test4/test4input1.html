<section id="input1" class="level1">
<h1>input1</h1>
<section id="subtitle" class="level2">
<h2>subtitle</h2>
<p><em>emphasis</em> <strong>bold</strong></p>
</section>
</section>
